include/deal.II/grid/grid_refinement.h

```diff
@@ -146,14 +146,14 @@ namespace GridRefinement
    * indicator. The default value of this argument is to impose no limit on
    * the number of cells.
    */
-  template <int dim, class Vector, int spacedim>
+  template <int dim, class VectorType, int spacedim>
   void
-  refine_and_coarsen_fixed_number (
-    Triangulation<dim,spacedim> &tria,
-    const Vector                &criteria,
-    const double                top_fraction_of_cells,
-    const double                bottom_fraction_of_cells,
-    const unsigned int          max_n_cells = std::numeric_limits<unsigned int>::max());
+  refine_and_coarsen_fixed_number
+  (Triangulation<dim,spacedim> &tria,
```

Comment:
Any reason why you changed the line the bracket is on? Is it consistent with what is done elsewhere?